src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -3208,7 +3202,9 @@ public static void createMissingStaticSitePins(BELPin belPin, SiteInst si, Cell
         Net net = (netOnSiteWire != null) ? netOnSiteWire : si.getDesign().getVccNet();
         if (net.isStaticNet()) {
             // SRL16Es that have been transformed from SRLC32E require GND on their A6 pin
-            if (cell.getType().equals("SRL16E") && siteWireName.endsWith("6")) {
+            if (cell.getType().equals("SRL16E") && siteWireName.endsWith("6") &&
+                    //
```

Comment:
Missing thought?

Reply:
This is a workaround for the symptom in https://github.com/Xilinx/RapidWright/pull/754 that shouldn't have slipped in here. Good spot!